### ISSUE TITLE
mask to Segs is not working well after last update

### ISSUE THREAD

**Issue by amir84ferdos** [ltdrdata/ComfyUI-Impact-Pack#633]
if you connect the segs preview node to mask segs it can not show the preview 

Error occurred when executing SEGSPreview:

The size of tensor a (1651) must match the size of tensor b (1717) at non-singleton dimension 2

File "C:\Tutorial\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Tutorial\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Tutorial\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Tutorial\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\segs_nodes.py", line 380, in doit
stack_image(cropped_image, cropped_mask)
File "C:\Tutorial\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\segs_nodes.py", line 340, in stack_image
image *= torch.tensor(mask)[None, ..., None]

**Comment by ltdrdata** [comment]
What is your workflow?
Especially the input resolution is important.

**Comment by amir84ferdos** [comment]
Sorry, it was my fault, but now everything is working well.
